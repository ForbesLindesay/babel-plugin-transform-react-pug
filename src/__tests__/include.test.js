import testHelper from './test-helper';

describe('without extname', () => {
  testHelper(__dirname + '/include.input.js');
});

describe('with extname', () => {
  testHelper(__dirname + '/include-extname.input.js');
});

describe('with space', () => {
  testHelper(__dirname + '/include-space.input.js');
});
