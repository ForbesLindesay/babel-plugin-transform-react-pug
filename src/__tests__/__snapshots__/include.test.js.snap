// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`without extname JavaScript output: transformed source code 1`] = `
"module.exports = <div>include success</div>;
"
`;

exports[`without extname html output: generated html 1`] = `
<div>
  include success
</div>
`;

exports[`without extname static html output: static html 1`] = `"<div>include success</div>"`;

exports[`with extname JavaScript output: transformed source code 1`] = `
"module.exports = <div>include success</div>;
"
`;

exports[`with extname html output: generated html 1`] = `
<div>
  include success
</div>
`;

exports[`with extname static html output: static html 1`] = `"<div>include success</div>"`;

exports[`with space JavaScript output: transformed source code 1`] = `
"module.exports = (
  <div>
    <div>include success</div>
  </div>
);
"
`;

exports[`with space html output: generated html 1`] = `
<div>
  <div>
    include success
  </div>
</div>
`;

exports[`with space static html output: static html 1`] = `"<div><div>include success</div></div>"`;
